src/workloads/tpch/denormalized/10/Q17.yml

```diff
@@ -0,0 +1,25 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/product-query"
+Description: |
+  Run TPC-H query 17 against the denormalized schema for scale 10.
+
+Clients:
+  Default:
+    QueryOptions:
+      socketTimeoutMS: -1
+
+Actors:
```

Comment:
Sadly, yes. This file/dummy actor template needs to stick around, otherwise the current test control will fail/complain that either the file for scale 10 Q17 doesn't exist or that the workload doesn't generate a file that it needs.

There is no way to optionally run this workload without creating a new/separate test control for scale 10. I don't want to do this because:
- This would result in code duplication between the test controls, doubling any maintenance work
- This would make it trickier to run scale 10 in a solo patch for an experiment (you would have to modify DSI and Genny rather than just the Genny workload to run this query on scale 10 in evergreen)
- Re-enabling this query is really easy, just uncomment the Genny Actor definition. Same goes for if we want to disable a query.